menu/navigation.json

```diff
@@ -1391,6 +1387,16 @@
                 ],
                 "label": "API/CLI",
                 "slug": "api-cli"
+              },
+              {
+                "items": [
+                  {
+                    "label": "Configuring your Load Balancer",
+                    "slug": "configuring-load-balancer"
+                  }
+                ],
+                "label": "Reference Content",
```

Comment:
```suggestion
                "label": "Reference content",
```

?? dunno

Reply:
All the other ones are in title case too, so unless we change them all it's easier just to keep this one the way it is ;-)